BeeSwift/AppDelegate.swift

```diff
@@ -133,8 +133,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         assert(Thread.isMainThread, "updateBadgeCount must be run on the main thread")
 
         let context = ServiceLocator.persistentContainer.viewContext
-        guard let goals = ServiceLocator.goalManager.staleGoals(context: context) else { return }
```

Comment:
Another approach would be to execute another code block on sign out, setting the badge count directly to 0 without first looking up goals in the db.

Reply:
No strong opinions either way. But if doing it this way, let's add a comment explaining it is also intended to cover the logged out case.